core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -58,19 +65,86 @@ object ZkUtils extends Logging {
   val IsrChangeNotificationPath = "/isr_change_notification"
   val EntityConfigPath = "/config"
   val EntityConfigChangesPath = "/config/changes"
+  
+  
+  def create(zkUrl: String, sessionTimeout: Int, connectionTimeout: Int, loginConfigFile: String): ZkUtils = {
```

Comment:
This is typically named `apply` in Scala and it means you can create an instance as `ZkUtils(...)`